src/components/Welcome.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Welcome({ items }) {
+  //if(!items){
```

Comment:
Would be good if there's a note why is this commented out. Temporary fix? Remove in next release? What should future devs know when they see this?